etc/config/c++.amazon.properties

```diff
@@ -3697,12 +3697,40 @@ libs.xtl.versions.053.path=/opt/compiler-explorer/libs/xtl/0.5.3/include
 libs.xtl.versions.0416.version=0.4.16
 libs.xtl.versions.0416.path=/opt/compiler-explorer/libs/xtl/0.4.16/include
 
+libs.ztdcuneicode.name=ztd.cuneicode
+libs.ztdcuneicode.url=https://github.com/soasis/cuneicode
+libs.ztdcuneicode.description=Robust C-style text encoding library<br /><br />Note that to use, you must add the "ztd.idk" library
+libs.ztdcuneicode.versions=main
+libs.ztdcuneicode.versions.main.version=main
+libs.ztdcuneicode.versions.main.path=/opt/compiler-explorer/libs/ztdcuneicode/main/include
+
+libs.ztdencodingtables.name=ztd.encoding_tables
+libs.ztdencodingtables.url=https://github.com/soasis/encoding_tables
+libs.ztdencodingtables.description=Collection of encoding tables for C and C++ programs to handle most table-driven and other styles of single and multi-byte encodings<br /><br />Note that to use, you must add the "ztd.idk" library
+libs.ztdencodingtables.versions=main
+libs.ztdencodingtables.versions.main.version=main
+libs.ztdencodingtables.versions.main.path=/opt/compiler-explorer/libs/ztdencodingtables/main/include
+
+libs.ztdidk.name=ztd.idk
+libs.ztdidk.url=https://github.com/soasis/idk
+libs.ztdidk.description=Industrial Development Kit (IDK) for core utilities (bit functions, comparison functions, and more) found across software engineering disciplines
+libs.ztdidk.versions=main
+libs.ztdidk.versions.main.version=main
+libs.ztdidk.versions.main.path=/opt/compiler-explorer/libs/ztdidk/main/include
+
+libs.ztdstaticcontainers.name=ztd.static_containers
+libs.ztdstaticcontainers.url=https://github.com/soasis/static_containers
+libs.ztdstaticcontainers.description=Fixed-size and similar containers library<br /><br />Note that to use, you must add the "ztd.idk" library
+libs.ztdstaticcontainers.versions=main
+libs.ztdstaticcontainers.versions.main.version=main
+libs.ztdstaticcontainers.versions.main.path=/opt/compiler-explorer/libs/ztdstaticcontainers/main/include
+
 libs.ztdtext.name=ztd.text
 libs.ztdtext.url=https://github.com/soasis/text
```

Comment:
also when you've added the other staticlinklibs, the complicated bit here is to add `libs.ztdtext.dependencies=lib1:lib2:lib3`
in order of what the linker can handle, so the one with the least exports on the left, the one that wants all the symbols on the end

Reply:
(where lib1 etc is the name of the .a file -lib, not the CE name for it)

Reply:
(if you add the dependencies to the other libs we auto-order the dependencies, that might also be a good option)

Reply:
Oh, okay! I'll add that too.